src/components/Logo/index.js

```diff
@@ -6,9 +6,9 @@ import styled from 'styled-components';
 import configuration from 'shared/services/configuration/configuration';
 
 const StyledLogo = styled.img`
-  width: ${props => props.tall === true ? configuration.logo.width : configuration.logo.smallWidth};
-  height: ${props => props.tall === true ? configuration.logo.height : configuration.logo.smallHeight};
-  max-height: ${props => props.tall === true ? '56px' : '32px'};
+  width: ${({ tall }) => tall ? configuration.logo.width : configuration.logo.smallWidth};
+  height: ${({ tall }) => tall ? configuration.logo.height : configuration.logo.smallHeight};
+  max-height: ${({ tall }) => tall === true ? '56px' : '32px'};
```

Comment:
```
max-height: ${({ tall }) => tall ? '56px' : '32px'};
```

Reply:
Removed explicit **true**, will add linting rules for this soon®